cookiecutter.json

```diff
@@ -9,9 +9,9 @@
     "wsgi_max_request_body_size": "1073741824",
     "wsgi_clear_untrusted_proxy_headers": false,
 
+    "environment_CHAMELEON_CACHE": "{{ cookiecutter.location_clienthome }}/cache",
```

Comment:
I like the idea of not using a dict here, but a namespace like thing.
IMO then it should be done this way for all with prefix `environment_` in the same way

Reply:
I agree